package.json

```diff
@@ -47,6 +47,7 @@
     "ngx-uploader": "2.2.0",
     "normalize.css": "4.2.0",
     "reflect-metadata": "0.1.9",
+    "roboto-fontface": "^0.7.0",
```

Comment:
We need to update yarn lock file. Set "roboto-fontface": "0.7.0" and run `yarn` command.